backend/app/views/spree/admin/orders/confirm/_shipment.html.erb

```diff
@@ -11,11 +11,11 @@
   <table class="stock-contents index" data-hook="stock-contents">
     <thead>
       <th colspan="2">
-        <%= Spree.t(:item_description) %>
+        <%= Spree::LineItem.human_attribute_name(:description) %>
```

Comment:
My way of looking at it is the table is showing the shipment but it's describing a bunch of line_items that make up the shipment